Knapsack: Make Bob a Sherpa (#2470)
exercism/problem-specifications: exercises/knapsack/introduction.md

Reviewer: As an ethnic group, `sherpa` should be cased to `Sherpa` throughout. I'd also recommend specifying the individual is a Sherpa guide and not just a Sherpa to avoid implying all Sherpas are guides.

# Introduction

> Bob is a thief.

Reviewer 2: We should mention he is a sherpa before mentioning an expedition.

> As he is a sherpa, he will be paid the value he carried to the base camp.

Reviewer 3: ```suggestion
As he is a sherpa, his payment will be equal to the value he carries to the base camp.
```

> After months of careful planning, the expedition Bob works for is about to leave.
> As he is a sherpa, he will be paid the value he carried to the base camp.

Reviewer 4: ```suggestion
Bob is a [sherpa][sherpa].
After months of careful planning, the expedition Bob works for is about to leave.
He will be paid the value he carried to the base camp.
```

In front of him are many items, each with a value and weight.

> Bob would gladly take all of the items, but his knapsack can only hold so much weight.

Reviewer 3: ```suggestion
Bob would gladly take all of the items, but his knapsack can only hold so much weight.

[sherpa]: https://en.wikipedia.org/wiki/Sherpa_people#Mountaineering
```

# Introduction

After months of careful planning, the expedition Bob works for is about to leave.
As he is a sherpa, he will be paid the value he carried to the base camp.
Lhakpa is a [Sherpa][sherpa] mountain guide and porter.
After months of careful planning, the expedition Lhakpa works for is about to leave.
She will be paid the value she carried to the base camp.

In front of him are many items, each with a value and weight.
Bob would gladly take all of the items, but his knapsack can only hold so much weight.

> Bob has to carefully consider which items to take so that the total value of his selection is maximized.

Reviewer 5: I've just noticed that this last sentence was removed. Was that on purpose?

Author: Yes, it is mostly repeated in the first sentence of `instructions.md`: `Your task is to determine which items to take so that the total value of her selection is maximized, taking into account the knapsack's carrying capacity.`
